### PR TITLE
changed request payment button text and tab description

### LINKED ISSUE #14482: Better name for "Request Payment" button
From IRC:
```
<gwillen> gmaxwell: if the button said 'receive payment' as you typed in your response, it wouldn't be quite as unintuitive
<gwillen> what it actually says is 'request payment'
<gmaxwell> gwillen: thanks! 
<sipa> let's fix that :)
...
<sipa> perhaps instead of "It is recommended to use a new receiving address for each transaction" it should say "Use the request payment button in the receive tab to create new addresses."
```
These simple GUI changes may help avoid a lot of users' confusion. A good issue for first-time contributors and for Hacktoberfest

### PR DESCRIPTION
**Pull request by HatboyWonder**
Small change by altering the text values for the request payment button and it's tab description to the ones suggested in https://github.com/bitcoin/bitcoin/issues/14482.

Closes #14482

### CODE REVIEW

**Comment by jonasschnelli** [comment]
utACK 59a9bd6d5895f0a2d96b505b73ed0ef1060c919c.
Though we should probably discuss how we could remove the address book and eventually increase the usability of per-transaction comments and labels.

**Comment by MeshCollider** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/14484/commits/59a9bd6d5895f0a2d96b505b73ed0ef1060c919c

@jonasschnelli agreed, this is a good first step for usability though

**Comment by HatboyWonder** [comment]
I can surely help with improving per-transaction comments and labels if you point me in the right direction.

File: src/qt/forms/receivecoinsdialog.ui [hunk 1]
```diff
@@ -108,7 +108,7 @@
              </size>
             </property>
             <property name="text">
-             <string>&amp;Request payment</string>
+             <string>&amp;Receive payment</string>
```
Comment (luke-jr) [inline src/qt/forms/receivecoinsdialog.ui]: This doesn't seem literally correct. Receiving only happens when the sender sends. :/

File: src/qt/addressbookpage.cpp [hunk 2]
```diff
@@ -107,7 +107,7 @@ AddressBookPage::AddressBookPage(const PlatformStyle *platformStyle, Mode _mode,
         ui->newAddress->setVisible(true);
         break;
     case ReceivingTab:
-        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. It is recommended to use a new receiving address for each transaction."));
+        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the request payment button in the receive tab to create new addresses."));
```
Comment (luke-jr) [inline src/qt/addressbookpage.cpp]: If the button is getting renamed, it doesn't make sense to refer to the old name here.

**Review by luke-jr — CHANGES_REQUESTED** [review]

File: src/qt/addressbookpage.cpp (hunk 2 above)
Comment (HatboyWonder) [reply src/qt/addressbookpage.cpp]: Agreed

File: src/qt/forms/receivecoinsdialog.ui (hunk 1 above)
Comment (HatboyWonder) [reply src/qt/forms/receivecoinsdialog.ui]: Should this text even change than. As far as I understand it's function it makes sense.

Comment (flack) [reply src/qt/forms/receivecoinsdialog.ui]: maybe call it something like "Create new receiving address" instead?

Comment (HatboyWonder) [reply src/qt/forms/receivecoinsdialog.ui]: Will do.

File: src/qt/addressbookpage.cpp [hunk 3]
```diff
@@ -107,7 +107,7 @@ AddressBookPage::AddressBookPage(const PlatformStyle *platformStyle, Mode _mode,
         ui->newAddress->setVisible(true);
         break;
     case ReceivingTab:
-        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the request payment button in the receive tab to create new addresses."));
+        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the 'Create new receiving address' button in the receive tab to create new addresses."));
```
Comment (luke-jr) [inline src/qt/addressbookpage.cpp]: Nit: Prefer to have the button label here as a `%1` token, so translations don't need to change when/if we rename the button again. (OTOH, the button itself needs `&amp;`, so maybe not worth the trouble...)

File: src/qt/forms/receivecoinsdialog.ui [hunk 4]
```diff
@@ -108,7 +108,7 @@
              </size>
             </property>
             <property name="text">
-             <string>&amp;Receive payment</string>
+             <string>&amp;Create new receiving address</string>
```
Comment (luke-jr) [inline src/qt/forms/receivecoinsdialog.ui]: Please keep `r` as the accelerator key.

**Comment by promag** [comment]
I think we shouldn't rush as this doesn't look consistent:

screenshot 2018-10-15 at 13 54 23

I agree with @jonasschnelli, first discuss, mock etc, and them implement?

**Comment by HatboyWonder** [comment]
> I think we shouldn't rush as this doesn't look consistent:
> 
> screenshot 2018-10-15 at 13 54 23
> 
> I agree with @jonasschnelli, first discuss, mock etc, and them implement?

Ok so what is the scope of these changes. Can this be done in the course of this issue?

**Comment by laanwj** [comment]
This seems to have bled out a bit.

**Comment by jonasschnelli** [comment]
I think this is a general improvement. We can do further extensions/cleanup later. Lets merge this.
utACK c46a603d7a149efa1e7af596e5437156d26742ac
@HatboyWonder: can you squash the second commits since its basically overwriting your first commit.

**Comment by fanquake** [comment]
@jonasschnelli squashed/rebased in #15829.

**Pull request closed without merging**